org-roam-capture.el

```diff
@@ -113,28 +113,60 @@ applies.
   "The Org-roam templates used during a capture from the roam-ref protocol.
 Details on how to specify for the template is given in `org-roam-capture-templates'.")
 
-(defun org-roam-capture--get (keyword)
-  "Gets the value for KEYWORD from the `org-roam-capture-template'."
-  (plist-get (plist-get org-capture-plist :org-roam) keyword))
-
-(defun org-roam-capture--put (&rest stuff)
-  "Puts properties from STUFF into the `org-roam-capture-template'."
+(defun org-roam-capture--get (keyword &optional local)
+  "Get the value for KEYWORD from the `org-roam-capture-template'.
+When LOCAL is non-nil, get the local value for the current
+buffer."
+  (let ((p (if local
+               org-capture-current-plist
+             org-capture-plist)))
+    (plist-get (plist-get p :org-roam) keyword)))
+
+(defun org-roam-capture--get-local (keyword)
+  "Get the local value for KEYWORD from the `org-roam-capture-template'."
+  (org-roam-capture--get keyword t))
```

Comment:
Instead of a wrapper function, we could pass a descriptive symbol in calls to `org-roam-capture--get`.
e.g.
```emacs-lisp
(org-roam-capture--get :capture-fn 'local)
```

Reply:
Sounds good to me.